docs/sources/installation/helm/concepts.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Helm Chart Components
+menuTitle: Helm Chart Components
+description: A short introduction to the components installed with the Loki Helm Chart.
+aliases:
+  - /docs/writers-toolkit/latest/templates/concept-template
```

Comment:
Another bug. @jdbaldry, this has the potential to become "far" reaching, so see previous comment.